metricbeat/logs/metricbeat

```diff
@@ -0,0 +1,10 @@
+2016-08-10T19:06:01-04:00 INFO Metrics logging every 30s
```

Comment:
The logs directory shuld not be commited. it is under .gitignore in the master branch